src/number-runtime.js

```diff
@@ -0,0 +1,57 @@
+define([
```

Comment:
The runtime library and the compiled formatters/parsers are two distinct things and should remain that way. You should load the runtime library once and could load multiple compiled formatters/parsers sequentially.